doc/modules/ROOT/pages/python-runtime/knowledge-graph-embeddings.adoc

```diff
@@ -0,0 +1,296 @@
+`loss_function_kwargs` can be used to set the `adversarial_temperature` and `margin` parameters.
+
+
+=== Optimizer
+
+Any pytorch optimizer can be used for training the model.
+To use non-default optimizer, specify the optimizer class name as a string in the `optimizer` parameter.
+All optimizer parameters except `params` can be passed as `optimizer_kwargs`.
+
+
+=== Negative sampling
+
+The loss function requires negative samples to compute the loss.
+The number of negative samples per positive sample is controlled by the `negative_sampling_size` parameter.
+When `use_node_type_aware_sampler` is set to `True`, negative nodes are sampled with the same label as the corresponding positive node.
+With or without node type awareness, the negative samples are sampled uniformly at random from the graph.
+
+
+=== Learning rate scheduler
+
+Any pytorch learning rate scheduler can be used for training the model.
+To use non-default learning rate scheduler, specify the scheduler class name as a string in the `lr_scheduler` parameter.
+All scheduler parameters except `optimizer` can be passed as `lr_scheduler_kwargs`.
+
+
+=== Inner normalisation
+
+In the original `TransE` paperfootnote:[Bordes, Antoine, et al. "Translating embeddings for modeling multi-relational data." Advances in neural information processing systems 26 (2013).]
+in `Algorithm 1`, line 5, the entity embeddings are normalized to have `Lp` norm of 1.
+Value of `p` is set by the `p_norm` parameter.
+For some datasets, this normalization might not be beneficial.
+To avoid this normalization, set `inner_norm` to `False`.
+
+
+=== Filtered metrics
+
+When we evaluate (compute metrics) the model, we compute scores for all possible triples with the same head or tail and relationship type as the test triple.
+Ranking the test triple among other triples is used for computing metrics, such as Mean Rank, Mean Reciprocal Rank, and Hit@k.
+
+When `filtered_metrics` is set to `False`, the ranking is done among all possible triples.
+
+When `filtered_metrics` is set to `True`, the ranking is done among only the triples that are not present in the training set.
+
+[[algorithms-embeddings-kge-syntax]]
+== Syntax
+
+[source, python, role=no-test]
+----
+gds.model.train(G,
+    num_epochs = 10,
+    embedding_dimension = 100,
+)
+----
+
+.Parameters
+[cols="1m,1m,1m,1", options="header"]
+|====
+| Parameter | Type | Default value | Description
+
+| num_epochs
+| int
+| N/A
+| Number of epochs for training (must be greater than 0)
+
+| embedding_dimension
+| int
+| N/A
+| Dimensionality of the embeddings (must be greater than 0)
+
+| epochs_per_checkpoint
+| int
+| max(num_epochs / 10, 1)
+| Number of epochs between checkpoints (must be greater than or equal to 0)
+
+| load_from_checkpoint
+| Optional[tuple[str, int]]
+| None
+| Checkpoint to load from, specified as a tuple (path, epoch)
```

Comment:
I forget how it works.
```suggestion
| Checkpoint to load from, specified as a tuple (modelname, epoch)
```
I think it's just the model name right?

Reply:
Yes